docs/resources/share.md

```diff
@@ -87,7 +85,7 @@ The following arguments are required:
 
 ### object Configuration Block
 
-* `name` (Required) - Full name of the object, e.g. `catalog.schema.name` for a tables, volumes and models, or `catalog.schema` for schemas.
+* `name` (Required) - Full name of the object, e.g. `catalog.schema.name` for a tables, volumes and models, or `catalog.schema` for schemas. Change forces creation of a new resource.
```

Comment:
Note: Since we now use name for hashing, we should create new resource on name change.